portable/MemMang/heap_4.c

```diff
@@ -411,7 +411,15 @@ static void prvHeapInit( void ) /* PRIVILEGED_FUNCTION */
 
     /* To start with there is a single free block that is sized to take up the
      * entire heap space, minus the space taken by pxEnd. */
+    #if defined( __clang__ )
```

Comment:
Can we keep the compiler specific changes out of the common files? Is it possible to suppress this warning at the file level in the application CMakeFile?

Reply:
Yes it is possible however it means that the check is disabled for the entire target and/or the entire heap_4.c file instead of just this one file location.  Moved to CMakeFile as part of freertos_kernel target.

Reply:
So, can this be removed now?

Reply:
Yes removed - updated list in description to include the cast-align in cmake.